src/dvclive/utils.py

```diff
@@ -192,8 +192,12 @@ def wrapper(*args, **kwargs):
 
 
 def rel_path(path, dvc_root_path):
-    absolute_path = Path(path).absolute()
-    return str(Path(os.path.relpath(absolute_path, dvc_root_path)).as_posix())
+    if dvc_root_path:
+        absolute_path = Path(path).absolute()
+        path = os.path.relpath(absolute_path, dvc_root_path)
+    if os.name == "nt":
+        return str(PureWindowsPath(path).as_posix())
```

Comment:
Convert windows-style paths to unix

Reply:
Was it handled before, caught by tests, or a separate fix along the way?

Reply:
It was caught by CI tests on Windows. It only impacts posting to Studio in Windows without a repo, so it is new to this scenario. Moved this logic outside of `rel_path`.